src/cms/templates/imprint/imprint_revisions.html

```diff
@@ -40,9 +52,9 @@ <h1 class="heading">{% trans 'Imprint revisions' %}</h1>
         <label class="inline-block mb-2 font-bold">{% trans 'Status' %}:</label>
```

Comment:
Here as well, could you add `data-editor="{{ imprint_translation.creator.profile.full_user_name }}"` to the previous line?
```
    <div class="w-full hidden revision-wrapper" id="revision-{{ imprint_translation.version }}"
         data-date="{{ imprint_translation.last_updated }}"  data-editor="{{ imprint_translation.creator.profile.full_user_name }}">
```